test/fixtures/linter/autofix/ansi/004_indentation/after.sql

```diff
@@ -9,20 +9,23 @@ WITH audience_counts AS (
         list_emails.active != 'D'
     GROUP BY
         user_id,
-        list_id)
+        list_id
+)
```

Comment:
I'm curious about this change. I tend to prefer fewer lines as long as things don't get too cluttered. (So I can see more code on the screen without scrolling.)

Reply:
So specific case with brackets I think is right, it ensures that opening bracket are at the same indent as their closing token.